src/client/websocket/WebSocketConnection.js

```diff
@@ -423,6 +417,15 @@ class WebSocketConnection extends EventEmitter {
     });
   }
 
+  /**
+   * Average heartbeat ping of the websocket, obtained by averaging the {@link WebSocketConnection#pings} property
+   * @type {number}
+   * @readonly
+   */
+  get ping() {
```

Comment:
How do we access this now? Before, `client.ping`, now..? Wouldn't we need to go (assuming a msg in a guild) `msg.guild.shard.ping`? Either then, shard is private, and thus shouldn't be accessed.

I presume `client.ping` being a getter that does some math across all pings would work., otherwise dunno